apps/system/utils/utils_ps.c

```diff
@@ -77,16 +77,10 @@ static const char *utils_statenames[] = {
 	"INACTIVE",
 	"WAITSEM ",
 	"WAITUNBLOCK",
-#ifndef CONFIG_DISABLE_SIGNALS
```

Comment:
Please make a separate commit for this file. 
All files in os and lib folders can be made into single commit. 
But for any other folders, we need to make separate commit.